awx/playbooks/project_update.yml

```diff
@@ -47,6 +82,7 @@
             force: "{{ scm_clean }}"
             track_submodules: "{{ scm_track_submodules | default(omit) }}"
             accept_hostkey: "{{ scm_accept_hostkey | default(omit) }}"
+          environment: "{{ git_environment | default({}) }}"
```

Comment:
I was trying to do something similar in https://github.com/ansible/awx/pull/14190, just to link it.